environment/integration/helpers.go

```diff
@@ -34,6 +36,46 @@ func init() {
 	})))
 }
 
+// createTestTempDir creates a temporary directory with a shorter path on Windows
+func createTestTempDir(t *testing.T, prefix string) string {
+	var dir string
+	var err error
+
+	// Get test name and replace slashes with underscores to avoid path separator issues
+	testName := t.Name()
+	testName = strings.ReplaceAll(testName, "/", "_")
+
+	if runtime.GOOS == "windows" {
```

Comment:
nit again: prefer build directives. it's quite hard to see what this fn does on either OS-group